scanomatic/data_processing/pheno/save.py

```diff
@@ -138,6 +140,13 @@ def save_state_to_zip(
     state: PhenotyperState,
     target: Optional[str] = None,
 ) -> Optional[BytesIO]:
+    JsonizerStateWriter = Callable[[BufferedWriter, Any], int]
+    PickleStateWriter = Callable[[BufferedWriter, Any], None]
```

Comment:
Since i guess we don't really care about return values, they just happen to be something, then maybe this could be a single `Callable[[BufferedWriter, Any], Any]`? Then it could be used for the `save_functions` variable below too.

Reply:
fixed